src/js/Item.ts

```diff
@@ -250,7 +250,7 @@ export class Item<Model extends ModelAttributes> {
         const img = this.document.createElement('img');
         img.setAttribute('src', this.model.thumbnailSrc);
 
-        this._image.style.backgroundImage = 'url(' + this.model.thumbnailSrc + ')';
+        this._image.appendChild(img);
```

Comment:
again, PS recommended html markup (https://photoswipe.com/data-sources/#custom-html-markup)  
`<img>` here is required for zoom transition (nice thumbnail pic to full page pic animation)  

as docs say it could be also done via `domItemData` but simply using <img> seems way cleaner